gv-user-network@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=gvisor-tap-vsock Network Traffic Forwarder
```

Comment:
This file is in the VM created in `createdisk.sh` with `tee /etc/systemd/system/gv-user-network@.service <<TEE ...`, is this file used?

Reply:
yes, it's not needed. Removed